data/containers/autoyast_containers.xml.ep

```diff
@@ -210,8 +210,10 @@
       <package>acl</package>
       % if ($check_var->('INSTALL_RUNTIMES', '1')) {
       <package>docker</package>
-      % unless ($check_var->('ARCH', 'aarch64') and $get_var->('VERSION') =~ /12-SP3|12-SP4|12-SP5|15$/) {
+      % if (!$check_var->('ARCH', 'aarch64') and $get_var->('VERSION') =~ /15-SP1|15-SP2|15-SP3|15-SP4|15-SP5$/) {
```

Comment:
More readable:

```suggestion
      % if (!$check_var->('ARCH', 'aarch64') and $get_var->('VERSION') =~ /15-SP[1-5]$/) {
```

Reply:
That is indeed more readable. Amended and VRs restarted. Thanks for the suggestion!